src/usr/local/buildpack/tools/python.sh

```diff
@@ -32,6 +32,11 @@ if [[ -z "${tool_path}" ]]; then
 
   curl -sSfLo ${file} ${PYTHON_URL}/${TOOL_VERSION}/python-${TOOL_VERSION}-${CODENAME}-${ARCH}.tar.xz || echo 'Ignore download error'
 
+  apt_install \
+    build-essential \
+    libmysqlclient-dev \
+    ;
```

Comment:
This will not work as is, because some tests are already running without root, so they will now fail. We need some abstractions for installing base deps before we can add this.

See the other open issues for more infos